### ISSUE TITLE
Color contrast in alert ribbon yellow theme does not meet WCAG standard ratio of 7:1

### ISSUE THREAD

**Issue by knice** [ucsc/ucsc-2022#317]
![screenshot-2024-04-18-123415](https://github.com/ucsc/ucsc-2022/assets/36719/0efe2b3a-3090-4b2c-b634-dac79e11c229)
